core-services/ipi-deprovision/gcp.sh

```diff
@@ -29,10 +29,14 @@ echo "deprovisioning clusters with a creationTimestamp before ${gce_cluster_age_
 export CLOUDSDK_CONFIG=/tmp/gcloudconfig
 mkdir -p "${CLOUDSDK_CONFIG}"
 gcloud auth activate-service-account --key-file="${GOOGLE_APPLICATION_CREDENTIALS}"
+
+GCP_PROJECT=$(gcloud projects list --format 'value(projectId)' | tr -d '\n')
+echo "GCP project: ${GCP_PROJECT}"
```

Comment:
This [broke deprovisioning][1]:

```
Activated service account credentials for: [do-not-delete-ci-provisioner@openshift-gce-devel-ci.iam.gserviceaccount.com]
GCP project: openshift-dev-installeropenshift-gce-devel-ci
ERROR: (gcloud.compute.networks.list) Some requests did not succeed:
 - The resource 'projects/openshift-dev-installeropenshift-gce-devel-ci' was not found
```

Presumably because those creds have access to both `openshift-dev-installer` and `openshift-gce-devel-ci` projects.  Either fix the script to iterate over available projects, or fix the creds to only access a single project?

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ipi-deprovision-gcp/1387869129041514496